### ISSUE TITLE
Potential bug in Version 0.18.4 (08.03.2016)

### ISSUE THREAD

**Issue by ghost** [QuiteRSS/quiterss#1113]
I don't know if this has been resolved yet, so I'm just putting this here in case no one has addressed it yet upstream. While working on my localized packaged version, I found that one of my friends who writes null titles was having their feed duplicated each time my client queried for updates. Is it possible that because the title is null, the list is simply duplicated each time its updated? Thank you.

**Comment by genodeftest** [comment]
Can you please update and try again with the latest version?

Which platform are you using QuiteRSS on?

**Comment by ghost** [comment]
Sure. I will try to recreate the problem. I'm on CentOS 7 using the standard package. Sorry, just converting from Debian to Redhat.

**Comment by genodeftest** [comment]
On CentOS (as well as any other linux) you could install debug symbols and run QuiteRSS under valgrind. Having a title set to null and list duplication looks like an invalid memory access to me.

**Comment by ghost** [comment]
I've never used Valgrind, so I think I'll use this as an opportunity to learn it; however, I cannot address this until next week. I will follow up with the ticket then with what I find. Thank you.

**Comment by Funcy-dcm** [comment]
Is there a link to the news feed?

**Comment by ghost** [comment]
No. I can try duplicating the RSS feed. It shouldn't be difficult as it is simply a long sequence of empty titles.

**Comment by Funcy-dcm** [comment]
I need raw data to check
